tiered-storage/file-system/src/main/java/org/apache/bookkeeper/mledger/offload/filesystem/impl/FileSystemManagedLedgerOffloader.java

```diff
@@ -208,7 +224,10 @@ public void run() {
                     long end = Math.min(needToOffloadFirstEntryNumber + ENTRIES_PER_READ - 1,
                             readHandle.getLastAddConfirmed());
                     log.debug("read ledger entries. start: {}, end: {}", needToOffloadFirstEntryNumber, end);
+                    long startReadTime = System.nanoTime();
                     LedgerEntries ledgerEntriesOnce = readHandle.readAsync(needToOffloadFirstEntryNumber, end).get();
+                    long cost = System.nanoTime() - startReadTime;
```

Comment:
`try-catch` here?

Reply:
I think we don't need a `try-catch` block here, if there are a exception, record latency is meaningless